### PR TITLE
Added URL generation for accessing CDN

### PR DESCRIPTION
**Pull request by LewisWorton**

### CODE REVIEW

File: Application/client/src/services/memberService.js
Comment (liam-mh) [inline Application/client/src/services/memberService.js:1]: Can this be placed in its own file, say mediaService or something? Unsure if it being in memberService makes sense as it is to do with our media items

File: Application/api-gateway/middleware/forwardRequestMiddleware.js
Comment (liam-mh) [inline Application/api-gateway/middleware/forwardRequestMiddleware.js:1]: why was this default value changed?

**Review by liam-mh — CHANGES_REQUESTED** [review]

File: Application/api-gateway/middleware/forwardRequestMiddleware.js
Comment (LewisWorton) [reply Application/api-gateway/middleware/forwardRequestMiddleware.js:1]: Quick test I did, forgot to change it back before PR so did it on browser.

**Review by liam-mh — APPROVED** [review]
thanks

**Pull request merged**